pgtrigger/apps.py

```diff
@@ -6,7 +6,7 @@
 def install(**kwargs):
     import pgtrigger
 
-    pgtrigger.install()
+    pgtrigger.install(**kwargs)
```

Comment:
@arpitremarkable trying to wrap my head around this a bit more. I'm assuming the `post_migrate` signal sends the "using" parameter as a kwarg, right? And then it is passed on to `pgtrigger.install` here.

When you migrate multiple databases, do you simply call `manage.py migrate` or do you manually do `manage.py migrate database=db_name` for each database?

In the former case, I assume that the Django DB router passes through the proper database name for each model. In the latter, I assume that django passes the "database" argument as the "using" flag.

It would be nice if we could get some tests for this in our test suite. I can assist if you can clarify how this works